installer/sakura-common.iss

```diff
@@ -1,4 +1,4 @@
-﻿#if VER < EncodeVer(5,6,1)
+#if VER < EncodeVer(5,6,1)
```

Comment:
Did you remove the BOM?

Reply:
It is actually removed with this PR.

Is it necessary for the installer file to have the BOM? If it is so, then is the rule documented somewhere? 

https://github.com/sakura-editor/sakura/wiki#%E6%8F%90%E6%A1%88%E8%A6%81%E6%9C%9B%E7%AD%89

Reply:
The reason I ask this is I simply don't know if it's necessary or not.
Of course I agree to leave it as it is unless there is a good reason to do so.

Reply:
I don't know whether Inno Setup requires a BOM, but, in general, I think that the BOM should not be changed implicitly in a PR.

Reply:
https://jrsoftware.org/ishelp/index.php?topic=creatinginstallations
> Installations are created by means of scripts, which are ASCII or Unicode (UTF-8 encoded with a BOM) text files

It seems that a BOM is required.

Reply:
Thank you for the clarification!